src/documents/unity/grow/GrowCompete_GettingStarted.html.md

```diff
@@ -214,7 +214,7 @@ public class ExampleAssets : IStoreAssets {
 	/** Virtual Categories **/
 
 	public static VirtualCategory GENERAL_CATEGORY = new VirtualCategory(
-		"General", new List<string>(new string[] {SHIELD_GOOD.ID})
+		"General", new List<string>(new string[] {SHIELD_GOOD.ItemId})
```

Comment:
@vedi, I'm not sure that previous version is correct but to avoid mistakes from my poor SOOMLA experience, I'll rollback it.